admin-console/src/main/webapp/src/components/CreateEditModal.tsx

```diff
@@ -8,12 +8,10 @@ interface CreateEditModalProps {
     isEdit: boolean;
     showModal: boolean;
     applicationName?: string;
-    initialEventType?: EventType;
-    isLoading?: boolean;
+    initialEventType?: Partial<EventType>;
+    isLoading: boolean;
```

Comment:
Updated the types, made the isLoading required and the initialEventType a Partial